### PR TITLE
[guilib] fixes and additions

### PR DESCRIPTION
**Pull request by mkortstiege**
Make fadelabels read the scroll tag. Report @ http://forum.kodi.tv/showthread.php?tid=224545

### CODE REVIEW

**Comment by mkortstiege** [comment]
Ping. jenkins build this please

File: xbmc/guilib/GUIControlFactory.cpp [hunk 1]
```diff
@@ -1174,6 +1174,10 @@ CGUIControl* CGUIControlFactory::Create(int parentID, const CRect &rect, TiXmlEl
         labelInfo, scrollOut, timeToPauseAtEnd, resetOnLabelChange);
 
       ((CGUIFadeLabelControl *)control)->SetInfo(infoLabels);
+
+      // check whether or not a scroll tag was specified.
+      if (scrollValue != CGUIControl::FOCUS)
+        ((CGUIFadeLabelControl *)control)->SetScrolling(scrollValue == CGUIControl::ALWAYS ? true : false);
```
Comment (tamland) [inline xbmc/guilib/GUIControlFactory.cpp]: No need for the ?: part. `scrollValue == CGUIControl::ALWAYS` evaluates to exactly that

**Comment by tamland** [comment]
All good on the api bumping side;)

**Comment by MartijnKaijser** [comment]
should gui also have an api bump?

**Comment by mkortstiege** [comment]
> should gui also have an api bump?

@ronie ?

**Comment by mkortstiege** [comment]
Addressed @tamland's comment and added the gui api bump.

**Comment by mkortstiege** [comment]
Please don't merge yet. I'll add other guilib related commits so we can avoid all the api version bumping.

**Comment by mkortstiege** [comment]
jenkins build this please.

**Comment by mkortstiege** [comment]
jenkins build this please

**Comment by ronie** [comment]
fwitw i'd rather prefer separate PR's instead of lumping them all together.
if this is done just to make gui bumps easier, then we'll have to come up with a better way to handle them.

**Comment by mkortstiege** [comment]
@ronie, ok will split them up again (later today).

**Comment by mkortstiege** [comment]
Splitted em up in separate PRs.

**Pull request closed without merging**